components/pricing_modal/self_hosted_content.tsx

```diff
@@ -235,7 +214,11 @@ function SelfHostedContent(props: ContentProps) {
                         customButtonDetails={(!isPostSelfHostedEnterpriseTrial && isAdmin) ? (
                             trialButton()
                         ) : undefined}
-                        planDisclaimer={isPostSelfHostedEnterpriseTrial ? undefined : <StartTrialCaution/>}
```

Comment:
We probably still need the legal verbage to start a trial on both cloud and self-hosted. @annelieseklein @johndavidlugtu we need to include this in the design as well

Reply:
Heads up that the legal copy from this one can be used: https://www.figma.com/file/rWB5bbi3DvpmVwVMc2xfOH/Update-Plans?node-id=125%3A60793.

Note that we need to use the mattermost.com/pl/software-evaluation-agreement for this -- so we need to ping Marketing

Reply:
Added it back 
<img width="1444" alt="Screenshot 2022-07-28 at 13 46 28" src="https://user-images.githubusercontent.com/28563179/181487532-10bbba94-0e3a-450b-b35b-b5ae81bb3001.png">

Reply:
> Heads up that the legal copy from this one can be used: https://www.figma.com/file/rWB5bbi3DvpmVwVMc2xfOH/Update-Plans?node-id=125%3A60793.
> 
> Note that we need to use the mattermost.com/pl/software-evaluation-agreement for this -- so we need to ping Marketing

@johndavidlugtu this link `https://mattermost.com/pl/software-evaluation-agreement` is dead

Reply:
@annelieseklein thoughts on the design (see screenshot)?

@AGMETEOR feel free to ping in the Marketing channel or to Emily Dela Cruz directly

Reply:
@emdecr heads up on link above

Reply:
@johndavidlugtu @marianunez  Can def set up that redirect up - just to clarify: we want `/pl/software-evaluation-agreement` to go to https://mattermost.com/software-evaluation-agreement/ ?

Reply:
Yes that's correct